src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -72,10 +45,11 @@ ucs_status_t uct_rdmacm_md_query(uct_md_h md, uct_md_attr_t *md_attr)
 int uct_rdmacm_is_sockaddr_accessible(uct_md_h md, const ucs_sock_addr_t *sockaddr,
                                       uct_sockaddr_accessibility_t mode)
 {
+    uct_rdmacm_md_t *rdmacm_md = ucs_derived_of(md, uct_rdmacm_md_t);
     struct rdma_event_channel *event_ch = NULL;
     struct rdma_cm_id *cm_id = NULL;
     int is_accessible;
-    uct_rdmacm_md_t *rdmacm_md = ucs_derived_of(md, uct_rdmacm_md_t);
+    char *ip_str = ucs_alloca(ucs_max(INET_ADDRSTRLEN, INET6_ADDRSTRLEN));
```

Comment:
if you are going to print the address anyway, might as well convert it to string in the beginning of the function, and then pass "str" to all printing functions